File: ReserbizAPP.LIB/Dtos/NewAccountStatementDto.cs
using System;
using ReserbizAPP.LIB.Enums;

namespace ReserbizAPP.LIB.Dtos
{
    public class NewAccountStatementDto
    {
        public int ContractId { get; set; }
        public DateTime DueDate { get; set; }
        public float ElectricBill { get; set; }
        public float WaterBill { get; set; }
        public AccountStatementTypeEnum AccountStatementType { get; set; }
    }
}

File: Reserbiz/src/app/_dtos/new-account-statement.dto.ts
import { AccountStatementTypeEnum } from '../_enum/account-statement-type.enum';

import { IBaseDto } from '../_interfaces/ibase-dto.interface';

export class NewAccountStatementDto implements IBaseDto {
  public contractId: number;
  public dueDate: Date;
  public electricBill: number;
  public waterBill: number;
  public accountStatementType: AccountStatementTypeEnum;

  constructor() {
    this.contractId = 0;
    this.dueDate = new Date();
    this.electricBill = 0;
    this.waterBill = 0;
  }
}
